docs/Dockerfile

```
@@ -0,0 +1,17 @@
+FROM squidfunk/mkdocs-material:9.6.8
+
+# Set the working directory inside the container
+WORKDIR /docs
+
+# Copy the documentation files into the container
+COPY ./docs .
+COPY .git /docs/.git
+
+# Install additional MkDocs plugins if needed
+RUN pip install -r requirements.txt
```

Hadolint (2.12.0)

[warning] Avoid use of cache directory with pip. Use `pip install --no-cache-dir <package>`

(DL3042)